routes/resources-api.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * All routes for Resource Data are defined here
+ * Since this file is loaded in server.js into api/resources,
+ *   these routes are mounted onto /api/resoucres
+ * See: https://expressjs.com/en/guide/using-middleware.html#middleware.router
+ */
+
+const express = require('express');
+const router  = express.Router();
+const resourceQueries = require('../db/queries/resources');
+
+//api to get all resources
+router.get('/', (req, res) => {
+  resourceQueries.getAllResources()
+    .then(resources => {
+      res.json({ resources });
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: err.message });
+    });
+});
+
+//api to get resources only for the loggedIn user
+
+router.get('/users/:id', (req, res) => {
+ // req.session.userId = req.params.id;
```

Comment:
is that line needed for future?

Reply:
yes required so that in case we add login and have session objects, i can make use of session objects. In case we decide not to have login / sessions later, I will anyway remove dead code and console.logs while refactoring code at the end.